generate/create_detection_data.py

```diff
@@ -88,45 +91,48 @@ def create_detector_data(dataset_name, dataset_path, image_name, image, data):
 
```

Comment:
This needs to be done after the threading part is done so that multiple threads are not accessing the same file at once.

Reply:
Fixed